### ISSUE TITLE
Deep Medicine - Eric Topol

### ISSUE THREAD

**Issue by amsaha** [amsaha/my-reading-list#35]

**Comment by amsaha** [comment]
**TL;DR**
*AI/ML can increase accuracy and speed of diagnosis and take away a lot of the paperwork that doctors spend a disproportionately large amount of time doing. Hopefully, this freeing up will strengthen the doctor patient bond by once again allowing the doctor to have a deep connect with the patient.*

Most of the book does not throw any surprises to me because I am more or less conversant with the fact that ML can take away a lot of the grunt work from the practice of medicine. The book is basically a sequence of evidences to show that is indeed the case. 

The notable exception is the the last chapter "Deep Empathy", which though again is not something new to me, but is very reassuring to read. 

Since my dad was a doctor, I find this last chapter (as well as snippets all over the book) reiterating something that he used to constantly say, "A doctor needs to spend time with the patient". Of course he was from the previous generation, and I suppose most of them were like him :-) 

Here is to a future of most doctors feeling (and practicing) the same :-)
